tests/test_cmdline.py

```diff
@@ -1137,7 +1145,7 @@ def test_cmdline_on_ignored_undeclared_option(
                 # deps = ['.']
                 pyenvs = ['baz', 'xyzzy']
                 # ignore_undeclared = []
-                # ignore_unused = []
+                # ignore_unused = {sorted(list(default_ignore_unused))}
```

Comment:
```suggestion
                # ignore_unused = {sorted(DEFAULT_IGNORE_UNUSED)}
```

Reply:
Addressed in https://github.com/tweag/FawltyDeps/pull/365/commits/b6afa98689e4720130fad434d0ce13e2d2b44a26.